src/app/[locale]/class-based/(classwork)/components/common/details/AttachmentsViewSection.tsx

```
@@ -45,6 +47,7 @@ export default function AttachmentsSection({ attachments }: Props) {
     return (
         <div className="grid grid-cols-1 md:grid-cols-2 gap-4">
             {attachments?.map((file) => <AttachmentItemWithDownload attachment={file} />)}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)